dll/win32/shell32/wine/pidl.c

```diff
@@ -520,10 +520,6 @@ BOOL _ILHACKCompareSimpleIds(LPCITEMIDLIST pidltemp1, LPCITEMIDLIST pidltemp2)
         if (!_ILIsDrive(pidltemp1) || !_ILIsDrive(pidltemp2) || pdata1->u.drive.szDriveName[0] != pdata2->u.drive.szDriveName[0])
             return FALSE;
     }
-    else
-    {
-        return FALSE;
-    }
```

Comment:
I very much doubt that this is correct, it basically classifies everything it does not know as 'equal'